### PR TITLE
Added Andrew Wilkes' video/text tutorial platforms

### PR DESCRIPTION
**Pull request by willnationsdev**

### CODE REVIEW

File: community/tutorials.rst [hunk 1]
```diff
@@ -40,12 +40,14 @@ Video tutorials
...
 - `Gonkee <https://www.youtube.com/channel/UCJqCPFHdbc6443G3Sz6VYDw>`_ (2D and 3D, GDScript, Shaders).
 - `TheGuideKnight <https://www.youtube.com/playlist?list=PLYcTJBw32KtX2fvOdtO_fS3Nd6sxM71tn>`_ (2D, GDScript).
+- `GDScript Dude <https://www.youtube.com/channel/UCQs0i6vKISElM6mh7OzLouQn>`_ (GDScript).
 
 Text tutorials
 --------------
 
 - `Godot Recipes by KidsCanCode <http://godotrecipes.com/>`__
 - `Steincodes <https://steincodes.tumblr.com>`__
+- `GDScript website by Andrew Wilkes <https://gdscript.com>`__
```
Comment (Calinou) [inline community/tutorials.rst:50]:
```suggestion
- `GDScript website by Andrew Wilkes <https://gdscript.com>`__
- `Godot Recipes by KidsCanCode <http://godotrecipes.com/>`__
- `Steincodes <https://steincodes.tumblr.com>`__
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Comment by Calinou** [comment]
Cherry-picked to the `3.2` branch.

**Comment by willnationsdev** [comment]
@Calinou 
Hmm, I noticed that the changes in the latest version that mention the unofficial "Godot Tutorials" channel are not ported to the 3.2 branch. Namely, [this commit](https://github.com/godotengine/godot-docs/commit/f2b7965ceab178ef7638b78f1bd15171ffd3cb76) (since it contains references to other stuff too). Should I open another pull request for those links?

**Comment by Calinou** [comment]
@willnationsdev Sure, feel free to open a pull request against the `3.2` branch :slightly_smiling_face:

**Pull request merged**